policyengine_canada/parameters/gov/provinces/ns/benefits/benefits.yaml

```diff
@@ -0,0 +1,7 @@
+description: Nova Scotia list of benefits.
```

Comment:
```suggestion
description: Nova Scotia provides these benefits.
```

Reply:
> sorry for back and forth - think this is it

It helps a lot to finilize my understanding of the farmatting